bors.toml

```diff
@@ -1,5 +1,8 @@
 status = [
-  "continuous-integration/jenkins/branch"
+  "continuous-integration/jenkins/branch",
+  "Run smoke tests on macos-latest",
```

Comment:
:question: :clown_face: Is there a way to make this a pattern, or use some form of ID instead of the name? I assume not, but doesn't hurt to ask :smile:

Reply:
I don't think so. The bors docs are relatively unclear about referencing the Github Action jobs. They don't mention any more dynamic solutions, so I guess it's unavailable. We could consider contributing something like this if we have a good idea for it, and find it valuable.